### ISSUE TITLE
[Bug] Onboarding theme selection accessibility updates

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#7973]
Our onboarding theme selection has a radio button that is focused, then a button that is focused. Really this should be one atomic item. 

"Light theme" text and "dark theme" text should be part of the one focusable item (one for each)

We probably need to set both of those to focusable=false. 

The "automatic" button is not announced as a radio button. It should be part of the same radio group as the other theme buttons.

**Comment by sblatz** [comment]
This is a nice to have. If it a small effort, let's get this in.

**Comment by asadotzler** [comment]
This is also reported at #9722

**Comment by mcarare** [comment]
This issue is included in https://github.com/mozilla-mobile/fenix/issues/9722.
